2020-11-27 15:09:07,589 - root - INFO - main.py:48 - START SERVER
2020-11-27 15:09:08,257 - root - INFO - gateway_server.py:234 - 客户端(('192.168.0.201', 4370))已成功连接。。
2020-11-27 15:09:08,276 - root - INFO - gateway_server.py:234 - 客户端(('192.168.0.117', 23))已成功连接。。
2020-11-27 15:09:08,777 - root - INFO - gateway_server.py:278 - 客户端(('192.168.0.97', 26))已成功连接。。
2020-11-27 15:21:19,295 - root - ERROR - Object2.py:58 - 'int' object has no attribute 'hex'
2020-11-27 15:25:06,160 - root - ERROR - Object2.py:58 - 'int' object has no attribute 'hex'
2020-11-27 15:25:51,187 - root - ERROR - Object2.py:58 - 'int' object has no attribute 'hex'
2020-11-27 15:37:36,254 - root - ERROR - Object2.py:58 - 'int' object has no attribute 'hex'
2020-11-27 15:39:17,208 - root - ERROR - Object2.py:58 - 'int' object has no attribute 'hex'
2020-11-27 15:42:42,308 - root - ERROR - Object2.py:58 - 'int' object has no attribute 'hex'
2020-11-27 15:44:15,257 - root - ERROR - Object2.py:58 - 'int' object has no attribute 'hex'
2020-11-27 15:48:57,187 - root - ERROR - Object2.py:58 - 'int' object has no attribute 'hex'
2020-11-27 15:51:10,329 - root - ERROR - Object2.py:58 - 'int' object has no attribute 'hex'
2020-11-27 16:02:20,936 - root - INFO - main.py:51 - START SERVER
2020-11-27 16:55:53,725 - root - INFO - main.py:51 - START SERVER
2020-11-27 16:57:58,017 - root - INFO - main.py:51 - START SERVER
2020-11-27 16:58:53,576 - root - INFO - main.py:51 - START SERVER
2020-11-27 17:17:17,739 - root - INFO - main.py:52 - START SERVER
2020-11-27 17:17:18,398 - root - INFO - gateway_server.py:235 - 客户端(('192.168.0.201', 4370))已成功连接。。
2020-11-27 17:17:18,402 - root - INFO - gateway_server.py:235 - 客户端(('192.168.0.117', 23))已成功连接。。
2020-11-27 17:17:18,552 - root - INFO - gateway_server.py:279 - 客户端(('192.168.0.97', 26))已成功连接。。
2020-11-27 17:45:39,414 - root - INFO - main.py:59 - START SERVER
2020-11-27 17:45:39,477 - root - ERROR - gateway_server.py:73 - getsockaddrarg: AF_INET address must be tuple, not str
2020-11-27 17:46:50,240 - root - INFO - main.py:61 - START SERVER
2020-11-27 17:46:50,299 - root - ERROR - gateway_server.py:73 - getsockaddrarg: AF_INET address must be tuple, not str
2020-11-27 17:49:20,815 - root - INFO - main.py:62 - START SERVER
2020-11-27 17:49:20,873 - root - ERROR - gateway_server.py:73 - getsockaddrarg: AF_INET address must be tuple, not str
2020-11-27 17:49:44,408 - root - INFO - main.py:62 - START SERVER
2020-11-27 17:49:45,053 - root - INFO - gateway_server.py:235 - 客户端(('192.168.0.201', 4370))已成功连接。。
2020-11-27 17:49:45,057 - root - INFO - gateway_server.py:235 - 客户端(('192.168.0.97', 23))已成功连接。。
2020-11-27 17:49:45,317 - root - INFO - gateway_server.py:282 - 客户端(('192.168.0.97', 26))连接创建线程失败。。
2020-11-27 17:52:58,609 - root - INFO - main.py:62 - START SERVER
2020-11-27 17:52:59,257 - root - INFO - gateway_server.py:235 - 客户端(('192.168.0.201', 4370))已成功连接。。
2020-11-27 17:52:59,261 - root - INFO - gateway_server.py:235 - 客户端(('192.168.0.97', 23))已成功连接。。
2020-11-27 17:53:00,150 - root - INFO - gateway_server.py:282 - 客户端(('192.168.0.97', 26))连接创建线程失败。。
2020-11-27 17:54:44,783 - root - INFO - main.py:60 - START SERVER
2020-11-27 17:54:45,434 - root - INFO - gateway_server.py:235 - 客户端(('192.168.0.201', 4370))已成功连接。。
2020-11-27 17:54:45,438 - root - INFO - gateway_server.py:235 - 客户端(('192.168.0.97', 23))已成功连接。。
2020-11-27 17:54:46,022 - root - INFO - gateway_server.py:282 - 客户端(('192.168.0.97', 26))连接创建线程失败。。
2020-11-27 17:57:31,424 - root - INFO - main.py:60 - START SERVER
2020-11-27 17:57:32,071 - root - INFO - gateway_server.py:235 - 服务端(('192.168.0.201', 4370))已成功连接。。
2020-11-27 17:57:32,074 - root - INFO - gateway_server.py:235 - 服务端(('192.168.0.97', 23))已成功连接。。
2020-11-27 17:57:32,595 - root - INFO - gateway_server.py:279 - 客户端(('192.168.0.97', 26))已成功连接。。
2020-11-27 18:02:15,198 - root - ERROR - Object2.py:58 - 'int' object has no attribute 'hex'
2020-11-27 18:15:08,218 - root - ERROR - Object2.py:58 - 'int' object has no attribute 'hex'
2020-11-27 18:33:42,020 - root - INFO - main.py:60 - START SERVER
2020-11-27 18:33:42,679 - root - INFO - gateway_server.py:235 - 服务端(('192.168.0.201', 4370))已成功连接。。
2020-11-27 18:33:42,682 - root - INFO - gateway_server.py:235 - 服务端(('192.168.0.97', 23))已成功连接。。
2020-11-27 18:33:42,686 - root - INFO - gateway_server.py:235 - 服务端(('192.168.0.117', 23))已成功连接。。
2020-11-27 18:33:43,427 - root - INFO - gateway_server.py:279 - 客户端(('192.168.0.97', 26))已成功连接。。
2020-11-27 18:34:46,246 - root - ERROR - Object2.py:58 - 'int' object has no attribute 'hex'
2020-11-27 18:36:18,337 - root - ERROR - Object2.py:58 - 'int' object has no attribute 'hex'
2020-11-27 18:37:37,311 - root - ERROR - Object2.py:58 - 'int' object has no attribute 'hex'
2020-11-27 18:38:10,236 - root - ERROR - Object2.py:58 - 'int' object has no attribute 'hex'
2020-11-27 18:44:12,211 - root - ERROR - Object2.py:58 - 'int' object has no attribute 'hex'
2020-11-27 18:47:00,251 - root - ERROR - Object2.py:58 - 'int' object has no attribute 'hex'
2020-11-27 18:53:05,192 - root - ERROR - Object2.py:58 - 'int' object has no attribute 'hex'
2020-11-27 19:06:59,453 - root - INFO - main.py:60 - START SERVER
2020-11-27 19:06:59,523 - root - ERROR - gateway_server.py:73 - __init__() missing 1 required positional argument: 'storeroom_id'
2020-11-27 19:09:21,636 - root - INFO - main.py:60 - START SERVER
2020-11-27 19:09:22,288 - root - INFO - gateway_server.py:235 - 服务端(('192.168.0.201', 4370))已成功连接。。
2020-11-27 19:09:22,291 - root - INFO - gateway_server.py:235 - 服务端(('192.168.0.97', 23))已成功连接。。
2020-11-27 19:09:22,293 - root - INFO - gateway_server.py:235 - 服务端(('192.168.0.117', 23))已成功连接。。
2020-11-27 19:10:14,754 - root - INFO - main.py:60 - START SERVER
2020-11-27 19:10:15,406 - root - INFO - gateway_server.py:235 - 服务端(('192.168.0.201', 4370))已成功连接。。
2020-11-27 19:10:15,409 - root - INFO - gateway_server.py:235 - 服务端(('192.168.0.97', 23))已成功连接。。
2020-11-27 19:10:15,411 - root - INFO - gateway_server.py:235 - 服务端(('192.168.0.117', 23))已成功连接。。
2020-11-27 19:10:15,790 - root - INFO - gateway_server.py:280 - 客户端(('192.168.0.97', 26))已成功连接。。
2020-11-27 19:11:16,310 - root - ERROR - Object2.py:59 - 'int' object has no attribute 'hex'
